tests/functional/pv/test_qos_limits.py

```
@@ -0,0 +1,532 @@
+            log.info(
+                f"[{test_id}] Executing active write & read I/O workload on volume..."
+            )
+            if volume_mode == constants.VOLUME_MODE_FILESYSTEM:
+                io_target = "/mnt/storage/test_io.img"
+            else:
+                io_target = "/dev/rbdblock"
+
+            # Execute sequential write I/O through Ceph-CSI mounted volume path
+            write_dd = (f"oc exec {pod_obj.name} -n {proj.namespace} -- "
+                        f"dd if=/dev/zero of={io_target} bs=1M count=20 conv=fsync status=progress")
+            write_res = exec_cmd(write_dd, shell=True, ignore_error=True)
```

Ruff (0.16.1)

[error] Function call with `shell=True` parameter identified, security issue

(S604)